### DISCUSSION TITLE
What is needed for `v1`?

### DISCUSSION THREAD

**Discussion by ansh** [enesozturk/react-native-hold-menu#108] [Q&A]
This package is great, however the README says 
> This package is experimental and still in progress. Using it in production is not preferred yet.

Which makes sense, there are still a good amount of bugs.

What is the roadmap for `v1`? Perhaps the community can contribute to it.

**Comment by NoodleOfDeath** [comment]
Happy to provide some contributions! I still use it in production with some fixes here and there
